src/Chapter02/Listing02.19.AssigningNullToAString.cs

```diff
@@ -13,8 +13,8 @@ public static void Main()
 
             #region EXCLUDE
             System.Console.WriteLine($"The age is: {age}");
-            #endregion
+            #endregion EXCLUDEr
```

Comment:
```suggestion
            #endregion EXCLUDE
```
remove r after EXCLUDE